functions/composer/composer_storage_trigger.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2019 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START composer_trigger]
+
+import google.auth
+import google.auth.app_engine
+import google.auth.compute_engine.credentials
+import google.auth.iam
+from google.auth.transport.requests import Request
+import google.oauth2.credentials
+import google.oauth2.service_account
+import requests
+import requests_toolbelt.adapters.appengine
+IAM_SCOPE = 'https://www.googleapis.com/auth/iam'
+OAUTH_TOKEN_URI = 'https://www.googleapis.com/oauth2/v4/token'
+
+
+def trigger_dag(data, context=None):
+    """Makes a POST request to the Composer DAG Trigger API
+
+    When called via GCF, data and context are Background function parameters.
+    For more info, refer to
+    https://cloud.google.com/functions/docs/writing/background#functions_background_parameters-python
+
+    When called via Python, a context object is not needed,
+    and a non-null data object must be present."""
+
+    # Fill in with your Composer info here
+    # Navigate to your webserver's login page and get this from the URL
+    # Or use the script found at
+    # https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/composer/rest/get_client_id.py
+    client_id = 'YOUR-CLIENT-ID'
+    # This should be part of your webserver's URL:
+    # {tenant-project-id}.appspot.com
+    webserver_id = 'YOUR-TENANT-PROJECT'
+    # The name of the DAG you wish to trigger
+    dag_name = 'composer_sample_trigger_response_dag'
+    webserver_url = 'https://' + webserver_id + \
```

Comment:
Nit: Prefer parentheses for multi-line statements. 

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces.

Also, the `+` operator should start the line.

https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

```suggestion
    webserver_url = (
        'https://'
        + webserver_id
        + '.appspot.com/api/experimental/dags/'
        + dag_name
        + '/dag_runs'
    )
```

Reply:
Ooo thank you for the references. I'm not as good with python style as I am Node!